pages/i18n/localization.json

```diff
@@ -32,12 +32,12 @@
             "header_bonus": "Bonus: easy migration"
           },
           "texts": {
-            "text_1": "Thanks to the open source code, you can modify and add CRM functions as you need them",
-            "text_2": "Configure the system in a few clicks without programming knowledge and receive orders from your website in a few minutes",
-            "text_3": "You have: an online store, courses or a web studio. <br /> <br /> We have: special templates that will save you time",
-            "text_4": "We know the specific needs of EdTech, agencies, and service providers — that's why we've created a CRM that's easy to use right away",
-            "text_5": "Connect your CMS and IP telephony in a few clicks. And for specific integrations, use Zapier, APIs, and public libraries",
-            "text_6": "Switch to Vilna in a few clicks with ready-made migration scripts from amoCRM and HubSpot"
+            "text_open_source": "Thanks to the open source code, you can modify and add CRM functions as you need them",
```

Comment:
no need to add "text" in the localization key, the parent property already says that you are in texts, update all of this everywhere